include/irr/video/IGPUObjectFromAssetConverter.h

```diff
@@ -576,7 +576,7 @@ inline created_gpu_object_array<asset::ICPURenderpassIndependentPipeline> IGPUOb
                 shaders[local_shdr_count++] = (*gpuShaders)[shdrRedirs[shdrIter++]].get();
 
         (*res)[i] = m_driver->createGPURenderpassIndependentPipeline(
-            nullptr, //make and use some driver's default pipeline cache
+            m_driver->getDefaultPipelineCache(),
```

Comment:
make it a parameter to the functions (also make a parameter struct to these functions of `IGPUObjectFromAssetConverter` for posteriority)